lucid/optvis/param/color.py

```diff
@@ -73,3 +73,16 @@ def to_valid_rgb(t, decorrelate=False, sigmoid=True):
     return tf.nn.sigmoid(t)
   else:
     return constrain_L_inf(2*t-1)/2 + 0.5
+
+def arbitrary_channels_to_rgb(*args, channels=None, **kwargs):
```

Comment:
I'm not sure I understand when I'd use this function in practice. Is it just for the tests? Maybe add a comment. :)

Reply:
Yes exactly :) It's a funky parametrization to test more than 3 channels

Reply:
Makes sense! Consider adding a brief comment, or even moving into the tests? Otherwise, LGTM!